build-template/NuGet.master.config

```diff
@@ -2,6 +2,5 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://nuget.org/api/v2/" />
-    <add key="AspNetMaster" value="https://www.myget.org/F/aspnetmaster/api/v2" />
```

Comment:
Yeah we definitely don't push *everything* to NuGet.org. Master will always be needed.